### PR TITLE
Integrate UI with API

### PR DESCRIPTION
**Pull request by tavisca-arupade**

### CODE REVIEW

File: src/app/services/product.service.ts [hunk 1]
```diff
@@ -29,7 +29,10 @@ export class ProductService implements IproductService {
     if (environment.isMockingEnabled) {
       return this.productMockService.AddProduct(product);
     } else {
-      return this.http.post<Product>(this.getUrl(environment.productSetting.addProductPath), product, {
+      let x = product;
```
Comment (achirametavisca) [inline src/app/services/product.service.ts]: remove this variable

File: src/environments/environment.ts [hunk 2]
```diff
@@ -1,15 +1,15 @@
 export const environment = {
   production: false,
   version: "/v1.0",
-  applicationName: "/ORP",
-  isMockingEnabled: true,
+  applicationName: "/OnlineRetailPortal",
+  isMockingEnabled: false,
```
Comment (achirametavisca) [inline src/environments/environment.ts]: set it true

**Review by achirametavisca — CHANGES_REQUESTED** [review]

File: src/app/services/product.service.ts (hunk 1 above)
Comment (tavisca-arupade) [reply src/app/services/product.service.ts]: done

File: src/environments/environment.ts (hunk 2 above)
Comment (tavisca-arupade) [reply src/environments/environment.ts]: done

**Review by achirametavisca — APPROVED** [review]
Look good.
Please merge.

**Pull request merged**